packages/cmrjs/index.js

```diff
@@ -1,174 +1,28 @@
-const fs = require('fs');
+'use strict';
+
 const got = require('got');
-const property = require('lodash.property');
 const { parseString } = require('xml2js');
-const log = require('@cumulus/common/log');
 const {
-  validate,
+  ingestConcept,
+  deleteConcept,
+  CMR
+} = require('./cmr-class');
+const {
   ValidationError,
   updateToken,
   getUrl,
   xmlParseOptions,
   getHost,
   hostId
 } = require('./utils');
+const {
+  getGranuleId,
+  getCmrFiles,
+  publish,
+  updateMetadata
+} = require('./cmr');
```

Comment:
If there's a better way to tease this apart and still export these functions, I'm all ears.
I don't have a better way to organize them at the moment.  (but I don't like that as I'm working on adding new functions I need to require and export them in my index.js)